packages/rune-games-cli/src/lib/validateGameFiles.ts

```diff
@@ -142,51 +139,23 @@ export async function validateGameFiles(
 
 async function validateMultiplayer({
   multiplayerValidationResult,
-  scripts,
   errors,
   logicJs,
-  indexHtml,
 }: {
   multiplayerValidationResult: NonNullable<ValidationResult["multiplayer"]>
-  scripts: HTMLElement[]
   errors: ValidationError[]
   logicJs: FileInfo | undefined
-  indexHtml: FileInfo
 }) {
-  const logicScript = scripts.find(
-    (script) =>
-      script.getAttribute("src") === "logic.js" ||
-      script.getAttribute("src")?.endsWith("/logic.js")
-  )
-
-  if (!logicScript) {
+  if (!logicJs) {
     errors.push({
-      message: "logic.js file is not included in index.html",
+      message: "logic.js must be included in the game files",
     })
   } else {
-    if (scripts.indexOf(logicScript) !== 1) {
-      errors.push({
-        message: "logic.js must be the second script in index.html",
-      })
-    }
-
-    if (!logicJs) {
-      errors.push({
-        message: "logic.js must be included in the game files",
-      })
-    } else {
-      if (path.dirname(indexHtml.path) !== path.dirname(logicJs.path)) {
-        errors.push({
-          message: "logic.js must be in the same directory as index.html",
```

Comment:
I think we want to keep this check

Reply:
Added it back in [9f0b98f](https://github.com/rune/rune-games-sdk/pull/146/commits/9f0b98f05761d238437819e67f0f8c2c8e8dfdad)